modules/images-configuration-allowed.adoc

```diff
@@ -64,11 +64,40 @@ The Machine Config Operator (MCO) watches the `image.config.openshift.io/cluster
 ifndef::openshift-rosa,openshift-dedicated[]
 .Verification
 
-* To check that the registries have been added to the policy file, use the following command on a node:
+* To check that the registries have been added to the policy file, enter the following command to obtain a list of your nodes:
 +
 [source,terminal]
 ----
-$ cat /host/etc/containers/policy.json
+$ oc get node
+----
++
+Example output
++
+[source,terminal]
+----
+NAME                                       STATUS   ROLES                  AGE   VERSION
+ci-ln-yjrspt2-72292-7jqm5-master-0         Ready    control-plane,master   37m   v1.27.8+4fab27b
+----
+
+. Run the following command to enter debug mode on the node:
```

Comment:
Replace the node name with the name from your own cluster, I assume?

Reply:
updating to <node_name>